data-acquisition/data-synthesizer/src/data_synthesizer/data_synthsizer.py

```diff
@@ -0,0 +1,159 @@
+import os
+import random
+import numpy as np
```

Comment:
Importing numpy for just random is definitely overkill. Why are we not using `random.choice` instead? Also, using:
```python
foo = np.random.choice(categories, 1)[0]
```
feels way more un-pythonic than:
```python
foo = random.choice(categories)
```